ext/date/php_date.c

```diff
@@ -2816,7 +2822,28 @@ PHP_METHOD(DateTimeImmutable, __construct)
 }
 /* }}} */
 
-/* {{{ proto DateTimeImmutable::createFromMutable(DateTimeZone object)
+/* {{{ proto DateTime::createFromImmutable(DateTimeImmutable object)
+   Creates new DateTime object from an existing immutable DateTimeImmutable object.
```

Comment:
Where exactly should I put it then? There are __construct for mutable & immutable together, __set_state for both together as well...